thrift/to_wire.go

```diff
@@ -137,6 +137,11 @@ func mapToValue(keySpec, valueSpec compile.TypeSpec, value interface{}) (wire.Ma
 
 	items := make([]wire.MapItem, 0, len(valueMap))
 	for k, v := range valueMap {
+		if v == nil {
+			// If no value is provided, then ignore this key.
```

Comment:
Do you think it's worth adding similar logic to `listToValue` as well?

Reply:
Hmm good question, it is nice to be consistent, but ignoring null list values has an impact on other elements (since they're ordered, it affects indexes). With maps/structs, ignoring a single field has no impact on other fields so it seems safer.

I could be convinced that the index of an item in a list isn't important though -- it's just the order that matters, and that is preserved.

Reply:
I don't feel strongly about supporting null-skipping for lists but
yeah, only the ordering matters. Indexes aren't sent over the wire.
And similar to the rest of this change: `null`/`nil` isn't a valid
value for list items either.

We can defer that to a future change if you're unsure about it.